src/Components/Error/Error.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from "react";
+import classNames from "classnames";
+
+type Props = {
+  error: string;
+  onClear: () => void;
+};
+
+export const Error: React.FC<Props> = ({ error, onClear }) => {
+  useEffect(() => {
+    const timeoutID = setTimeout(onClear, 3000);
+
+    return () => {
+      clearTimeout(timeoutID);
+    };
+  }, [error]);
+
+  return (
+    <div
+      className={classNames(
+        "notification is-danger is-light has-text-weight-normal",
+        { hidden: error.length === 0 }
+      )}
+    >
+      <button type="button" className="delete" onClick={onClear} hidden={false}>
+        {"delete "}
```

Comment:
```suggestion
        delete
`
is it work like here?

Reply:
if I remove the text from the button element it shows the error  A control must be associated with a text label  jsx-a11y/control-has-associated-label